content/docs/2.9/operate/prometheus.md

```diff
@@ -16,11 +16,30 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 
 ### Metrics Adapter
 
-The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`.  The metrics collected in the Metrics Adapter are only active when the HPA is active (> 0 replicas).
+The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`. The metrics collected in the Metrics Adapter are only active when the HPA is active (> 0 replicas).
 
 The following metrics are being gathered:
 
 - `keda_metrics_adapter_scaler_error_totals` - The total number of errors encountered for all scalers.
 - `keda_metrics_adapter_scaled_object_error_totals`- The number of errors that have occurred for each scaled object.
 - `keda_metrics_adapter_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_metrics_adapter_scaler_metrics_value`- The current value for each scaler's metric that would be used by the HPA in computing the target average.
+
+## Premade Grafana dashboard
+
+A premade [Grafana dashboard](https://github.com/kedacore/keda/tree/main/config/grafana/keda-dashboard.json) is available to visualize metrics exposed by the KEDA Metrics Adapter.
+
+![KEDA Grafana dashboard](/img/grafana-dashboard.png)
```

Comment:
I think the latest dashboard is different - Would you mind testing the latest template? If you don't have time then that's OK.

Reply:
For some reasons those metrics are not available in our environment:
- kube_horizontalpodautoscaler_status_current_replicas
- kube_horizontalpodautoscaler_spec_max_replicas

Reply:
This is coming from https://github.com/kubernetes/kube-state-metrics - You don't have it?

Reply:
Does it break the dashboard import or just lacks info in the dashboard? Maybe we can just merge without updating the image (assuming we remove info from 2.8)

Reply:
This graph will just be empty if the metric is missing. So no problem if you ask me.

Reply:
Great, let's proceed then. THanks!